src/css/transformTypography.ts

```diff
@@ -18,7 +18,8 @@ export function transformTypographyForCSS(
   }
 
   let { fontFamily, fontWeight, fontSize, lineHeight } = value;
-  fontWeight = transformFontWeights(fontWeight);
+
+  fontWeight = fontWeight !== undefined ? `${transformFontWeights(fontWeight)}` : undefined;
```

Comment:
Hm I actually recently changed things to where if a value is undefined, null or in the case of fontweight an empty string, I prefill the value with a default value of `400`, I'd like to keep that behavior.

Reply:
I think that's still the case? That logic happens a few lines down

```js
return `${isNothing(fontWeight) ? 400 : fontWeight} ${isNothing(fontSize) ? '16px' : fontSize}/${
    isNothing(lineHeight) ? 1 : lineHeight
  } ${isNothing(fontFamily) ? 'sans-serif' : fontFamily}`;
```

Also, there are some tests that cover this case, and they seem to succeed?

Reply:
This was needed because I was wrestling with Typescript here. I'm not really into Typescript, so maybe (probably) there's a better way.

Reply:
Ah I see yeah, I fixed the type issues for you :)

Reply:
Thanks!

This line was the missing link for me :) 
```js
fontFamily = hasWhiteSpace(fontFamily as string | undefined) ? `'${fontFamily}'` : fontFamily;
```